tests/end_to_end/testdata/esbuild-datadog-integration/template.yaml

```diff
@@ -0,0 +1,24 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+
+Parameters:
+  DataDogLayers:
+    Description: DataDog layers
+    Type: CommaDelimitedList
+    Default: "arn:aws:lambda:us-east-1:464622532012:layer:Datadog-Node16-x:93, arn:aws:lambda:us-east-1:464622532012:layer:Datadog-Extension:44"
```

Comment:
Can we add a comment here how to find latest layer information so we can keep it up to date?